### PR TITLE
Updated gsl

### PR DESCRIPTION
**Pull request by mmottl**

### CODE REVIEW

File: packages/gsl/gsl.1.18.2/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocamlfind" "remove" "gsl"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-bigarray"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```
Comment (avsm) [inline packages/gsl/gsl.1.18.2/opam]: This camlp4 line should just be `camlp4` -- the version constraints are set from the current compiler in the `camlp4` package description.

**Comment by mmottl** [comment]
@avsm Thanks, I've manually fixed this.

@Chris00 This incorrect line was generated by oasis2opam.  Could you please take a look at this issue?

**Comment by Chris00** [comment]
@mmottl [Done](https://github.com/ocaml/oasis2opam/commit/0267cf9dd2c3c48b3aca6cdf482d48636f4847f5).  Would you want a new release?

**Comment by mmottl** [comment]
@Chris00 Awesome, thanks!  No new release necessary, I've fixed the pull request manually.  I'll test your fix the next time I make a release.

**Comment by mmottl** [comment]
Oops, pull request should remain open.

**Pull request merged**